src/compute/src/sink/correction.rs

```diff
@@ -304,13 +312,22 @@ pub(crate) struct ConsolidatingVec<D> {
     /// A lower bound for how small we'll shrink the Vec's capacity. NB: The cap
     /// might start smaller than this.
     min_capacity: usize,
+    /// Denominator in the growth rate, where 2 corresponds to doubling, and `n` to `1 + 1/(n-1)`.
+    ///
+    /// If consolidation didn't free enough space, at least a linear amount, increase the capacity
+    /// The `slop` term describes the rate of growth, where we scale up by factors of 1/slop.
```

Comment:
```suggestion
    /// The `slop` term describes the rate of growth, where we scale up by factors of 1/(slop-1).
```

... right?

Reply:
Yes, but I think that term is *added* to the current capacity, so the factor is more like `slop/(slop - 1)`. I'll fix though!